base/server/python/pki/server/cli/plugin.py

```diff
@@ -0,0 +1,818 @@
+                    print("    Defer On Failure: {}".format(nest_keys['deferOnFailure']))
+                if nest_keys.get('fileName', None):
+                    print("    File name: {}".format(nest_keys['fileName']))
+                if nest_keys.get('keyAttributes', None):
+                    print("    Key Attributes: {}".format(nest_keys['keyAttributes']))
+                if nest_keys.get('dnpattern', None):
+                    print("    DN Pattern: {}".format(nest_keys['dnpattern']))
+                if nest_keys.get('bindDN', None):
+                    print("    Bind DN: {}".format(nest_keys['bindDN']))
+                if nest_keys.get('bindPWPrompt', None):
+                    print("    Bind PW Prompt: {}".format(nest_keys['bindPWPrompt']))
+                if nest_keys.get('clientcertNickname', None):
+                    print("    Client Cert Nick name: {}".format(nest_keys['clientcertNickname']))
+                if nest_keys.get('host', None):
+                    print("    Hostname: {}".format(nest_keys['host']))
+                if nest_keys.get('port', None):
+                    print("    Port: {}".format(nest_keys['port']))
+                if nest_keys.get('secureConn', None):
+                    print("    Secure Connection: {}".format(nest_keys['secureConn']))
+                if nest_keys.get('version', None):
+                    print("    Version: {}".format(nest_keys['version']))
+                if nest_keys.get('maxConn', None):
+                    print("    Max Connections: {}".format(nest_keys['maxConn']))
+                if nest_keys.get('minConn', None):
+                    print("    Min Connections: {}".format(nest_keys['minConn']))
+                if nest_keys.get('basedn', None):
+                    print("    Base DN: {}".format(nest_keys['basedn']))
+                if nest_keys.get('authType', None):
+                    print("    Auth Type: {}".format(nest_keys['authType']))
+                if nest_keys.get('ldapByteAttributes', None):
+                    print("    LDAP Bytes Attributes: {}".format(nest_keys['ldapByteAttributes']))
+                if nest_keys.get('ldapStringAttributes', None):
+                    print("    LDAP String Attributes: {}".format(
+                        nest_keys['ldapStringAttributes']))
+                if nest_keys.get('shrTokAttr', None):
+                    print("    Shared Token Attribute: {}".format(nest_keys['shrTokAttr']))
+                print()
+
+
+class AuthPluginManagerAddCLI(pki.cli.CLI):
+    def __init__(self, parent):
+        super(AuthPluginManagerAddCLI, self).__init__(
+            'add', 'Register new auth plugin manager'
+        )
+
+        self.parent = parent
+
+    def print_help(self):
+        print("Usages: pki-server %s-auth-plugin-manager-add [OPTIONS]"
+              % self.parent.parent.parent.name)
+        print()
+        print('  -t, --type                         Plugin type')
```

Comment:
The `pki-server ca-auth-manager-add` should take the manager ID from the command line arguments instead of `--type` option since it's the primary identifier for auth managers. It should be just like `pki-server ca-auth-manager-del`.